Dockerfile

```diff
@@ -0,0 +1,80 @@
+#
+# Docker image for console integration testing.
+#
+# We may also reuse it in CI to unify installation process and as a general binaries building
+# tool for production servers.
+#
+# Dynamic linking is used for librocksdb and libstdc++ bacause librocksdb-sys calls
+# bindgen with 'dynamic' feature flag. This also prevents usage of dockerhub alpine-rust
+# images which are statically linked and have guards against any dlopen. I would rather
+# prefer all static binaries so we may change the way librocksdb-sys builds or wait until
+# we will have our own storage and drop rockdb dependency.
+#
+
+#
+# build postgres separately -- this layer will be rebuilt only if one of
+# mentioned paths will get any changes
+#
+FROM alpine:3.13 as pg-build
```

Comment:
What is the reason of using two different alpine tags, ‘alpine:20210212’ and ‘alpine:3.13’? It looks like you did it intentionally, but it is not clear for me why

Reply:
I've used edge alpine in cases where I bring rocksdb package which is for edge. It seems to work okay on 3.13, but just to decrease amount of potential problems. However I accidentally used 3.13 instead of edge in resulting package -- will fix.